tests/integration/targets/meraki_mr_radio/tasks/main.yml

```diff
@@ -10,7 +10,7 @@
       auth_key: "{{ auth_key }}"
       state: present
       org_name: "{{ test_org_name }}"
-      net_name: IntTestNetworkWireless
+      net_id: '{{ test_net_id }}'
```

Comment:
This approach works okay. However, I'd rather we create a new network and use the network ID from there instead. Would you please update?

Reply:
All right! But I do not have permission to create a network. So I will not test it! I will do it today!

Reply:
You should be able to create a new organization and network for free.

Reply:
I can not add the new network in the Sandbox 😢 Can you test it ?